contracts/market/template/MarketCore.sol

```diff
@@ -599,10 +597,10 @@ contract MarketCore is AccessControlledAndUpgradeableModifiers, IMarketCommon, I
   */
 
   /// @notice Place the market in a state where no more price updates or mints are allowed
-  function _deprecateMarket() internal {
+  function _deprecateMarket() internal checkMarketNotDeprecated {
```

Comment:
No known bug here without this modifier - totally benign to call this function multiple times from what we see (ie it is idempotent). But why not make it so it can be called once. "Prudence"